From 282100ab4c39d6d6936be840a281b69775042190 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@nabladev.com>
Date: Mon, 7 Jul 2025 22:00:23 +0200
Subject: [PATCH 952/954] fix(mmc): increase MMC_DEFAULT_MAX_RETRIES

Increase MMC_DEFAULT_MAX_RETRIES to 50 to prevent early
CMD13 failure on Intelligent Memory eMMC devices.

Upstream-Status: Pending
Signed-off-by: Marek Vasut <marex@nabladev.com>
---
 drivers/mmc/mmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/mmc.c b/drivers/mmc/mmc.c
index b51e744364..248e356279 100644
--- a/drivers/mmc/mmc.c
+++ b/drivers/mmc/mmc.c
@@ -18,7 +18,7 @@
 #include <lib/utils.h>
 #include <plat/common/common_def.h>
 
-#define MMC_DEFAULT_MAX_RETRIES		5
+#define MMC_DEFAULT_MAX_RETRIES		50
 #define SEND_OP_COND_MAX_RETRIES	100
 
 #define MULT_BY_512K_SHIFT		19
-- 
2.53.0

